runtime/queries/beancount/highlights.scm

```diff
@@ -0,0 +1,35 @@
+(date) @variable.builtin
+(txn) @variable.builtin
+
+(account) @type
+
+[
+  (amount)
+  (incomplete_amount)
+  (amount_tolerance)
+  (number)
+] @constant.numeric
+
+
+(key) @label
```

Comment:
Let's use `@variable.other.member` which aligns to fields/properties in other languages

```scm
[(key_value) (key)] @variable.other.member
```

(including `key_value` will also fix the highlight for colons).

Reply:
This one should be good now.